_docs/node-operator-requirements-service.md

```diff
@@ -0,0 +1,104 @@
+
+![onfinality-setup-5](../assets/img/node-operator/onfinality-setup-5.png)
+
+### Step 4: Sync and observe your node
+Once your node is successfully deployed, you can find it in the "Dedicated Nodes" section of the site. Click on your node and observe the following:
+- Parachain blocks are syncing appropriately
+- Relay chain blocks are syncing appropriately
+- No configuration errors in the console log. You can find your console by clicking on "Console logs" on the top right hand. 
+- CPU, Memory, and Storage are within reasonable range of use.
+
+Wait and observe in this step until your node is fully synced to the relay chain and parachain. It's important that BOTH are synced. You can cross-reference the respective network blocks on the [PolkadotJS App](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.turing.oak.tech#/explorer).
+
+![onfinality-setup-6](../assets/img/node-operator/onfinality-setup-6.png)
+
+### Step 5: Rotate keys in your collator node
+Once your node is fully synced, the next step is to setup an association with the node and its peers. To do so you need to generate a session key. The steps are as follows.
+*Step 5.1: Grab the WSS address* 
+In the monitoring view of your node, click the "API Endpoints" button on the top right. You'll see a number of options. Stick with "RPC", and copy the "Websocket" address, starting with `wss://node-*`.
+
+![onfinality-setup-7](../assets/img/node-operator/onfinality-setup-7.png)
+*Step 5.2: Navigate to PolkadotJS' RPC call* 
+Take the copied Websocket address, and navigate to PolkadotJS. For ease of reference, [here](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/rpc) is the local development default URL.
+
+Paste the `wss://node-*` address under "custom endpoint" input field, and hit Enter or click "Switch" on the top of the panel. 
+
+![onfinality-setup-8](../assets/img/node-operator/onfinality-setup-8.png)
+
+*Step 5.3: RPC call to rotate keys* 
+Once your node's UI has loaded, then click on *Developer > RPC call* in the dropdown. Once your there your URL should look something like `https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fnode-12345-*#/rpc`. Please follow the following steps:
+- 5.3.1: Call RPC `author.rotateKeys`. It should return a 0x string.
+- 5.3.2: Very important! Save the 0x string safely. You will need this when registering as a collator. If you lose this, you can always re-run this call to grab the new session key.
+
+If you're managing multiple nodes, you can also double check that the session key is associated with the node by calling `author.hasSessionKeys()` and inserting the session key you believe it's associated with.
+
+![onfinality-setup-9](../assets/img/node-operator/onfinality-setup-9.png)
+
+### Step 6: Onboard your collator
+Once you're fully synced, you're ready to on-board as a collator. Proceed to this page to move forward and to start producing blocks. [Collator On-boarding](./collators.md)
```

Comment:
The `Collator On-boarding` link is not working for me.

Reply:
fixed